extensions/objects/templates/FilepathEditor.js

```diff
@@ -163,6 +188,46 @@ oppia.directive('filepathEditor', [
           $scope.clearScopeData();
         };
 
+        $scope.generateResampledImageFile = function() {
+          var dataURIToBlob = function(dataURI) {
+            // Convert base64/URLEncoded data component to raw binary data
+            // held in a string.
+            var byteString;
+            if (dataURI.split(',')[0].indexOf('base64') >= 0) {
+              byteString = atob(dataURI.split(',')[1]);
+            }
+            else {
+              byteString = unescape(dataURI.split(',')[1]);
```

Comment:
unescape() is deprecated. Any alternative? 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/unescape